templates/base.html

```diff
@@ -69,7 +69,9 @@
                 </a>
                 <div id="collapseThree" class="collapse" aria-labelledby="headingTwo" data-parent="#accordionSidebar">
                     <div class="bg-white py-2 collapse-inner rounded">
+                        {% if user.is_staff %}
```

Comment:
shouldn't the if come before the `div id="collapseThree"`?

Reply:
it's not possible to do that, because if I do that, then all collapse submenu is not going to be seeing for end users. And we are now adding a submenu on Tournaments that is going to be available for all users, so.... it's not possible to do what you ask, because then, end-users, are going to lose all possibility to access to Tournaments submenu (including what they have available)